quick-guide-for-developers.md

```diff
@@ -0,0 +1,54 @@
+
+# The Way We Work: Quick Guide for Developers
+
+## Your Role
+
+Your role is to use software engineering best practices to support rapid and safe delivery of high-quality software.
+
+## Your Focus
+
+At this stage in your career you should focus on your own personal success and professional development, while doing what you can to help others to succeed.
+
+## Your Responsibilities
+
+Your key responsibilities are to:
+
+- Carry out **Feature Development** work within a Delivery Team 
+- Complete **20% Time** work within each Delivery Team sprint
+- **Pair** with other members of a Delivery Team where this would be productive
+- **Buddy** with newer developers to help them settle into their role
+
+Details on how these responsibilities should be carried out can be found in The Way We Work.
+
+## How You Will Be Supported
+
+Your line manager will:
```

Comment:
Line manager: A line manager should:

or

Line manager: Your line manager is expected to:

or 

Your line manager is required to: